worldedit-core/src/main/java/com/sk89q/worldedit/command/tool/brush/ClipboardBrush.java

```diff
@@ -42,16 +43,30 @@ public class ClipboardBrush implements Brush {
     public ClipboardBrush(ClipboardHolder holder, boolean ignoreAirBlocks, boolean usingOrigin) {
         this.holder = holder;
         this.ignoreAirBlocks = ignoreAirBlocks;
+        this.ignoreStructureVoidBlocks = true;
```

Comment:
afaict, this is being defaulted to `true` for anyone creating their own clipboardbrush, but `false` for anyone creating a paste via pastebuilder. is there a reason for this mismatch?